pkg/proxy/ipvs/proxier_test.go

```diff
@@ -3120,7 +3125,7 @@ func Test_syncService(t *testing.T) {
 				t.Errorf("Case [%d], unexpected add IPVS virtual server error: %v", i, err)
 			}
 		}
-		if err := proxier.syncService(testCases[i].svcName, testCases[i].newVirtualServer, testCases[i].bindAddr); err != nil {
+		if err := proxier.syncService(testCases[i].svcName, testCases[i].newVirtualServer, testCases[i].bindAddr, sets.NewString()); err != nil {
```

Comment:
Would be nice if we can somehow add bindedAddrs as part of the test table but I see that this unit test doesn't really check for binded addresses anyways. Maybe something for a future PR.